site/content/docs/user/quick-start.md

```diff
@@ -135,13 +135,11 @@ kubectl apply -f my-manifest-using-my-image:unique-tag
 **Note**: The Kubernetes default pull policy is `IfNotPresent` unless
 the image tag is `:latest` in which case the default policy is `Always`.
 `IfNotPresent` causes the Kubelet to skip pulling an image if it already exists.
-If you want those images loaded into node to work as expected, please:
+If you want those images loaded into node to work as expected, please either:
 
-- don't use a `:latest` tag
-
-and / or:
-
-- specify `imagePullPolicy: IfNotPresent` or `imagePullPolicy: Never` on your container(s).
+- If you use a `:latest` tag set `imagePullPolicy` to `IfNotPresent` or `Never`
+- If you use a unique tag whenever your image changes either leave
```

Comment:
Thanks for the PR, but, I'm concerned that this doesn't discouragd `:latest` and conflates dealing with `:latest` and generally dealing with `imagePullPolicy` which should be two distinct but related topics, and more generally covered in the upstream documentation.

We should maybe breifly mention that you don't want `Always` above this, and link to the upstream docs again there.

The upstream docs recommend against latest, and they're right to do so, we should find a way to continue to suggest that here.

The "either or" should probably still be written along the lines of "please not use a :latest tag, or if you must, then use these settings".